### ISSUE TITLE
Unable to drag windows while BetterTouchTool is running

### ISSUE THREAD

**Issue by sarahbrazill** [folivoraAI/BetterTouchTool#2105]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at boastr.net@gmail.com instead.

---

#### Description of bug/feature request/question:

When BetterTouchTool is running:
-  I am unable to drag windows to rearrange them with both my mouse and my touchpad. 
-  I am unable to drag files from one folder to another. 
-  And I have been unable to highlight text in a window.

I didn't change my preferences, and saw no obvious preference settings that could cause this when I looked. 

#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):

MacBook Pro touchpad; Logitech bluetooth mouse

#### Device information:

Type of Mac: MacBook Pro (13-inch, 2017)
macOS version: High Sierra Version 10.13.3
BetterTouchTool version: 2.340

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.

**Comment by fifafu** [comment]
I have not heard of such an issue before. Have you already tried to restart your machine?

You could try to click the manage presets button in BTT and create a new empty profile to see whether it's some setting you have that causes this.

**Comment by sarahbrazill** [comment]
Yes, repeatedly. But I think I solved it:

It appears as though Apple's Ink application was conflicting with BetterTouchTool; once I disabled Ink, BTT starting working properly. 

Thank you for your quick response!

**Comment by fifafu** [comment]
That's interesting, thanks for reporting! I'll try to reproduce it here.

**Comment by sarahbrazill** [comment]
No problem.

FYI: (in case this assists in your reproduction) I have a Huion Q11K Tablet; the Ink application only appeared after the driver for the tablet was installed (screengrab included).  

And before I figured it out, I recorded a quick gif to show the odd behavior in BTT (also attached); every time you see the red circle, that's me attempting to click and drag, highlight, etc.

Hope that helps!

<img width="780" alt="screen shot 2018-02-12 at 2 16 30 pm" src="https://user-images.githubusercontent.com/28459143/36115220-69ea0ce6-1000-11e8-890b-1b52b779b59f.png">

![2018-02-12 14_22_59](https://user-images.githubusercontent.com/28459143/36115199-5b8228e6-1000-11e8-99b7-e44ca84e7659.gif)
